### ISSUE TITLE
xorgxrdp package missing and starting vncserver

### ISSUE THREAD

**Issue by yecin98** [neutrinolabs/xrdp#1109]
Hello guys,

I am running Kali linux on Windows 10 WSL and I am trying to run XFCE.
I followed the tutorial on the bottom of this page [https://www.kali.org/news/kali-linux-in-the-windows-app-store/](url) but something is going wrong as I am getting an error when connecting.
I have seen a similar  issue here [https://github.com/neutrinolabs/xrdp/issues/1101](url) and I tryied to follow.
The first time I installed tigervnc it worked. But when I tried to run it again I get this error
```
vncserver: /usr/bin/Xtigervnc did not start up, please look into '/root/.vnc/DESKTOP-I7N9UKT.localdomain:1.log' to determine the reason! -1
Killing Xtigervnc process ID 367... success!
```
Any help would be very appreciated. I'm new to this.

**Comment by metalefty** [comment]
Did you read the log? Not just copy & pasting.
Why don't you show us '/root/.vnc/DESKTOP-I7N9UKT.localdomain:1.log' ?

**Comment by yecin98** [comment]
Yes, I did read it. Didn't find any clue. Maybe it's me, again, I'm new to this stuff.
Here's the log
`Xvnc TigerVNC 1.7.0 - built Dec  4 2017 09:39:19
Copyright (C) 1999-2016 TigerVNC Team and many others (see README.txt)
See http://www.tigervnc.org for information on TigerVNC.
Underlying X server release 11905000, The X.Org Foundation


Sat Apr 21 22:48:56 2018
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on local interface(s), port 5901
 vncext:      created VNC server for screen 0

Xvnc TigerVNC 1.7.0 - built Dec  4 2017 09:39:19
Copyright (C) 1999-2016 TigerVNC Team and many others (see README.txt)
See http://www.tigervnc.org for information on TigerVNC.
Underlying X server release 11905000, The X.Org Foundation


Sat Apr 21 22:50:31 2018
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on local interface(s), port 5901
 vncext:      created VNC server for screen 0

Xvnc TigerVNC 1.7.0 - built Dec  4 2017 09:39:19
Copyright (C) 1999-2016 TigerVNC Team and many others (see README.txt)
See http://www.tigervnc.org for information on TigerVNC.
Underlying X server release 11905000, The X.Org Foundation


Sun Apr 22 17:25:00 2018
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on local interface(s), port 5901
 vncext:      created VNC server for screen 0`

**Comment by dretax** [comment]
Same shit going on, yet the issue is just simply closed. Nice.

**Comment by yecin98** [comment]
I was not the one to close it. Then again, I still can't find a solution and no one is helping. Sorry.
Please update us if you find a solution.

**Comment by dretax** [comment]
I saw that It wasn't you.

**Comment by adamwbb** [comment]
same issue here with Ubuntu 20.04 on WSL
`hdrecovery@TROWDSHDRECOVERY:~$ vncserver -geometry 1600x900 :0 -v
/usr/bin/Xtigervnc :0 -desktop TROWDSHDRECOVERY.localdomain:0 (hdrecovery) -auth /home/hdrecovery/.Xauthority -geometry 1600x900 -depth 24 -rfbwait 30000 -rfbauth /home/hdrecovery/.vnc/passwd -rfbport 5900 -pn -localhost -SecurityTypes VncAuth
vncserver: /usr/bin/Xtigervnc did not start up, please look into '/home/hdrecovery/.vnc/TROWDSHDRECOVERY.localdomain:0.log' to determine the reason! -1
Killing Xtigervnc process ID 8589... success!
hdrecovery@TROWDSHDRECOVERY:~$ cat /home/hdrecovery/.vnc/TROWDSHDRECOVERY.localdomain:0.log

Xvnc TigerVNC 1.10.0 - built Apr  9 2020 06:49:31
Copyright (C) 1999-2019 TigerVNC Team and many others (see README.rst)
See https://www.tigervnc.org for information on TigerVNC.
Underlying X server release 12008000, The X.Org Foundation


Thu Feb 18 13:27:35 2021
 vncext:      VNC extension running!
 vncext:      Listening for VNC connections on local interface(s), port 5900
 vncext:      created VNC server for screen 0

Thu Feb 18 13:27:41 2021
 Connections: accepted: 127.0.0.1::52936
 SConnection: Client needs protocol version 3.8
 SConnection: Client requests security type VncAuth(2)
 VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
 VNCSConnST:  Client pixel format depth 6 (8bpp) rgb222

Thu Feb 18 13:28:05 2021
 VNCSConnST:  closing 127.0.0.1::52936: Server shutdown
 EncodeManager: Framebuffer updates: 3
 EncodeManager:   RRE:
 EncodeManager:     Solid: 24 rects, 1.4576 Mpixels
 EncodeManager:            408 B (1:3573.25 ratio)
 EncodeManager:   Total: 24 rects, 1.4576 Mpixels
 EncodeManager:          408 B (1:3573.25 ratio)
 ComparingUpdateTracker: 0 pixels in / 0 pixels out
 ComparingUpdateTracker: (1:-nan ratio)`

**Comment by metalefty** [comment]
It is TigerVNC issue rather than xrdp. We xrdp developer are not responsible on that.

**Comment by metalefty** [comment]
One advice: Try uninstaling xrdp completely from your system and just install TigerVNC. And then run `vncserver -geometry 1600x900 :0 -v`. If the issue still exists, it is definitely TigerVNC issue and xrdp project is not the place to report that issue.
